crates/lang/codegen/src/generator/metadata.rs

```diff
@@ -84,16 +84,16 @@ impl Metadata<'_> {
         quote! {
             ::ink_metadata::ContractSpec::new()
                 .constructors([
-                    #(#constructors ,)*
+                    #( #constructors ),*
```

Comment:
Does this actually make a difference as far as `quote` is concerned, or is it an aesthetic preference?

Reply:
It does make a difference if the list of constructors is not empty.
Before: `.constructors([ first, second, ])`
Now: `.constructors([ first, second ])`
Note the missing comma at the end.

Reply:
🤦‍♂️ didn't notice that while skimming through